app/views/api/stripe_transaction_disputes/object_events/_base.json.jbuilder

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
```

Comment:
I don't believe this file has anything to do with the specific change.

Reply:
Done!